lib/bundler/cli/add.rb

```diff
@@ -2,23 +2,32 @@
 
 module Bundler
   class CLI::Add
-    def initialize(options, gem_name)
-      @gem_name = gem_name
+    def initialize(options, gems)
+      @gems = gems
       @options = options
       @options[:group] = @options[:group].split(",").map(&:strip) if !@options[:group].nil? && !@options[:group].empty?
     end
 
     def run
+      # raise error when no gems are specified
+      raise InvalidOption, "Please specify gems to add." if @gems.empty?
+
+      # show warning when options are used with mutiple gems
+      # for time being they will be applied to the all gems
+      Bundler.ui.warn "All the options will be applied to all gems`."
```

Comment:
there's no need for this, discussed offline, it's how `gem install` works

Reply:
Yup removing it.